File: beecrowd_exercicios_em_Ruby/ad_hoc/escalator.rb
n = gets.to_i

while n != 0
  total = 10

  m = gets.chomp

  if n > 1

    m =  m.split(' ').map(&:to_i)

    (0...(m.length - 1)).each do |x|

      total += if m[x + 1] - m[x] < 10
                 m[x + 1] - m[x]
               else
                 10
               end

    end

  end

  puts total

  n = gets.to_i

end

File: beecrowd_exercicios_em_python/ad-hoc/escalator.py
n = int(input())

while n != 0:

    total = 10

    m = input()

    if n > 1:
        m = list(map(int, m.split(" ")))

        for i in range(n - 1):
            if m[i + 1] - m[i] < 10:
                total += m[i + 1] - m[i]
            else:
                total += 10

    print(total)

    n = int(input())
